tests/Asset/Bar.php

```diff
@@ -2,7 +2,7 @@
 
 namespace League\Container\Test\Asset;
 
-class Bar
+class Bar implements BarInterface
```

Comment:
this doesn't have anything to do with it, btw. i can remove it again if necessary.